examples/create_arbitrary_rundir.py

```diff
@@ -0,0 +1,18 @@
+import fv3config
```

Comment:
In the current master of fv3config, there is a bash command for this called `write_run_directory`. Please either update to use that, or rename this file to `write_rundir.py` or `write_run_directory.py` (to be more consistent with the naming used by fv3config and by the example in fv3gfs-python).

Reply:
I've tried the write_run_directory subroutine in fv3config and I get errors in the configuration YAML file.  I guess you changed the required format.  I don't want to introduce more changes to tests that are working currently.

Reply:
renamed create_arbitrary_rundir.py to write_rundir.py